pkg/oidcclient/login.go

```diff
@@ -60,6 +60,11 @@ const (
 	defaultLDAPUsernamePrompt = "Username: "
 	defaultLDAPPasswordPrompt = "Password: "
 
+	// For CLI-based auth, such as with LDAP upstream identity providers, the user may use these environment variables
+	// to avoid getting interactively prompted for username and password.
+	defaultUsernameEnvVarName = "PINNIPED_USERNAME"
+	defaultPasswordEnvVarName = "PINNIPED_PASSWORD"
```

Comment:
Looks like this line needs a linter ignore:

```suggestion
	defaultPasswordEnvVarName = "PINNIPED_PASSWORD" //nolint:gosec // this is not a credential
```

Reply:
Fixed in 58ab57201fd32f07ca0c29deb03a48ce312d3b58